text-recognition/index.d.ts

```diff
@@ -62,9 +62,17 @@ interface ITextRecognition {
    * Recognize text in the image
    *
    * @param imageURL The URL/Path of the image to process
+   *
+   * @param language @optional The language to use for non-latin
+   * text recognition. The default is English. Available languages:
+   * - Chinese
+   * - Devanagari
+   * - Japanese
+   * - Korean
+   *
    * @returns Text recognition result
    */
-  recognize: (imageURL: string) => Promise<TextRecognitionResult>;
+  recognize: (imageURL: string, language?: string | null) => Promise<TextRecognitionResult>;
```

Comment:
I'd say let's make `language` an enum instead. See [@react-native-ml-kit/translate-text's `TranslateLanguage` enum](https://github.com/a7medev/react-native-ml-kit/blob/62ad4f3db2b083b8f527c36212ea7cc65a410cb8/translate-text/index.js#L9).

Reply:
Totally onboard with this change. Will address tonight 👍